.travis.yml

```diff
@@ -32,76 +32,79 @@ addons:
 env:
     global:
         # Modify these to represent the newest versions
-        - LATEST_NUMPY=1.10.2
-        - LATEST_PANDAS=0.17.1
-        - MATPLOTLIB_VERSION=">=1.5.1"
+        - PREVIOUS_PANDAS=0.15.2
         # Fixed global vars
         - PYTHON_VERSION=2.7
         - TEST_MODE='offline'
-        - NUMPY_VERSION="=$LATEST_NUMPY"
-        - PANDAS_VERSION="=$LATEST_PANDAS"
+        - NUMPY_VERSION='stable'
+        - MAIN_CMD='python setup.py'
+        - CONDA_CHANNELS='astropy astropy-ci-extras sunpy'
+        - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy matplotlib requests beautiful-soup sqlalchemy scikit-image pytest wcsaxes pyyaml pandas' # pandas last to change version number on env
+        - CONDA_ALL_DEPENDENCIES2="suds-jurko glymur $CONDA_ALL_DEPENDENCIES"
 
-    matrix:
-        - PYTHON_VERSION=2.7
+
+#    matrix:
+#        - PYTHON_VERSION=2.7 # egg info
 #        - PYTHON_VERSION=3.3
 #        - PYTHON_VERSION=3.4
 #        - PYTHON_VERSION=3.5
 
 matrix:
     include:
+         # Try MacOS X
+         - os: osx
+           env: PYTHON_VERSION=2.7 SETUP_CMD='test'
+               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES2
+
+         # Try on Linux
          - os: linux
-           env: NUMPY_VERSION="=1.9.3"
+           env: PYTHON_VERSION=2.7 SETUP_CMD='test'
+                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES2
          - os: linux
-           env: PANDAS_VERSION="=0.16.2"
+           env: PYTHON_VERSION=2.7 SETUP_CMD='test'
+                CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES2=$PREVIOUS_PANDAS" # all in quotes so it may take it
```

Comment:
@astrofrog I found a way!! ;)
Just double quote the whole variable set, so I can append the `=X.y` version to the package...